sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignature.java

```diff
@@ -413,6 +422,14 @@ public static TimestampParameter timestampParameter() {
       return TIMESTAMP_PARAMETER;
     }
 
+    public static SideInputParameter sideInputParameter(
+        TypeDescriptor<?> elementT, @Nullable String fieldAccessString) {
```

Comment:
shouldn't be called fieldAccessString. Maybe sideInputId?

Reply:
Done